setup.py

```diff
@@ -25,7 +25,7 @@
     license='MIT',
     install_requires=[
         'numpy', 'scikit-learn', 'pandas', 'torch>=1.0.0', 'diskcache', 'dill',
-        'selfies', 'upfp'
+        'selfies', 'upfp', 'SmilesPE==0.0.3'
```

Comment:
```suggestion
        'selfies', 'upfp', 'SmilesPE'
```
I assume something in version 0.0.4 was not working for you? If it is just broken and might be fixed in later versions I would stay lenient on the version restriction (in the install_requires). Or did the API change alot and we are sure not to allow any past/future versions?

Reply:
Interesting. I think the state of the repo has deviated from the `pypi` packge ([repo](https://github.com/XinhaoLi74/SmilesPE/blob/master/SmilesPE/__init__.py)). If you do a plain `pip install` you get `0.0.3`, this is why I fixed it to that one (wasn't even aware a newer one exists). But probably better to be lenient, I agree.

Reply:
I made all dependencies lenient `>=0.0.3` @C-nit.